hdfs_datanode/assets/configuration/spec.yaml

```
@@ -0,0 +1,30 @@
+name: HDFS Datanode
+files:
+  - name: hdfs_datanode.yaml
+    options:
+      - template: init_config
+        options:
+          - template: init_config/http
```

```suggestion
          - template: init_config/http
          - template: init_config/default
```

let's also include default template here